druid-derive/src/lens.rs

```diff
@@ -83,18 +83,24 @@ fn derive_struct(input: &syn::DeriveInput) -> Result<proc_macro2::TokenStream, s
     // Define lens types for each field
     let defs = fields.iter().filter(|f| !f.attrs.ignore).map(|f| {
         let field_name = &f.ident.unwrap_named();
-        let docs = format!(
-            "Lens for the field `{}` on [`{1}`](super::{1})",
+        let struct_docs = format!(
+            "Lens for the field `{}` on [`{1}`](super::{1}).",
+            field_name, ty
+        );
+
+        let fn_docs = format!(
+            "Creates a new lens for the field `{}` on [`{1}`](super::{1}). Use [`{1}::{0}`](super::{1}::{0}) instead.",
             field_name, ty
```

Comment:
can you name the format arguments, (for some reason, I am not able to suggest changes on two lines :thinking:)

Reply:
Sorry, I misinterpreted your feedback! I've pushed a fix. How's it looking now?